be-python-flask/Jenkinsfile.template

```diff
@@ -20,6 +20,7 @@ odsPipeline(
   ]
 ) { context ->
   stageBuild(context)
```

Comment:
hi @stitakis, just in case, in this stage there is also already a call to `junit`, see: https://github.com/opendevstack/ods-quickstarters/blob/7e5f08e8dce7371fab79129d2429c86c5c6d7dee/be-python-flask/Jenkinsfile.template#L46